code/datums/brain_damage/hypnosis.dm

```diff
@@ -0,0 +1,53 @@
+/datum/brain_trauma/hypnosis
+	name = "Hypnosis"
+	desc = "Patient's unconscious is completely enthralled by a word or sentence, focusing their thoughts and actions on it."
+	scan_desc = "looping thought pattern"
+	gain_text = ""
+	lose_text = ""
+	resilience = TRAUMA_RESILIENCE_SURGERY
+
+	var/hypnotic_phrase = ""
+	var/regex/target_phrase
+
+/datum/brain_trauma/hypnosis/New(phrase)
+	if(!phrase)
+		qdel(src)
+	hypnotic_phrase = phrase
+	try
```

Comment:
This code is directly ripped from tg, I assume it's done this way so that the trauma is removed if an incorrect string is set as the phrase? I'm not sure on how to change this honestly.

Reply:
I could try removing the try statement..? Lemme dig up the original PR.

Reply:
> Hypnosis code improvements

Hypnosis no longer runtimes forever if it can't generate a valid regex
from the user phrase

Hypnosis also no longer constantly regenerates the regex and instead
just matches using the existing regex

https://github.com/tgstation/tgstation/pull/42901